test: integration tests now against bazel built packages

alexeagle opened the pull request.

alexeagle committed:
test: integration tests now against bazel built packages

mary-poppins commented:
You can preview 5c1e98e at https://pr22810-5c1e98e.ngbuilds.io/.

mary-poppins commented:
You can preview a9833cd at https://pr22810-a9833cd.ngbuilds.io/.

mary-poppins commented:
You can preview 2518fab at https://pr22810-2518fab.ngbuilds.io/.

mary-poppins commented:
You can preview 6f20ede at https://pr22810-6f20ede.ngbuilds.io/.

alexeagle commented:
Handing this off to @IgorMinar who has already been updating the integration tests for changes in Angular 6

mary-poppins commented:
You can preview 9586c39 at https://pr22810-9586c39.ngbuilds.io/.

mary-poppins commented:
You can preview 8bb8772 at https://pr22810-8bb8772.ngbuilds.io/.

mary-poppins commented:
You can preview f3617c5 at https://pr22810-f3617c5.ngbuilds.io/.

mary-poppins commented:
You can preview 7581edf at https://pr22810-7581edf.ngbuilds.io/.

mary-poppins commented:
You can preview 1c939f8 at https://pr22810-1c939f8.ngbuilds.io/.

alexeagle commented:
This PR moves the tests to CircleCI at the same time as switching to Bazel outputs, which is hard to land. We can do it in two steps instead. Attempting to build the packages with Bazel on travis...

alexeagle commented:
... and I failed to get Travis to build all the packages in https://github.com/angular/angular/pull/23177 so I'll try this approach again

mary-poppins commented:
You can preview 5c1e98e at https://pr22810-5c1e98e.ngbuilds.io/.
You can preview a9833cd at https://pr22810-a9833cd.ngbuilds.io/.
You can preview 2518fab at https://pr22810-2518fab.ngbuilds.io/.
You can preview 6f20ede at https://pr22810-6f20ede.ngbuilds.io/.
You can preview 9586c39 at https://pr22810-9586c39.ngbuilds.io/.
You can preview 8bb8772 at https://pr22810-8bb8772.ngbuilds.io/.
You can preview f3617c5 at https://pr22810-f3617c5.ngbuilds.io/.
You can preview 7581edf at https://pr22810-7581edf.ngbuilds.io/.
You can preview 1c939f8 at https://pr22810-1c939f8.ngbuilds.io/.
You can preview ca9d3ad at https://pr22810-ca9d3ad.ngbuilds.io/.
You can preview e77c76f at https://pr22810-e77c76f.ngbuilds.io/.

mary-poppins commented:
You can preview 8f34e4b at https://pr22810-8f34e4b.ngbuilds.io/.

mary-poppins commented:
You can preview df3ea04 at https://pr22810-df3ea04.ngbuilds.io/.

mary-poppins commented:
You can preview 3690271 at https://pr22810-3690271.ngbuilds.io/.

mary-poppins commented:
You can preview 78ad61b at https://pr22810-78ad61b.ngbuilds.io/.

mary-poppins commented:
You can preview 2d89759 at https://pr22810-2d89759.ngbuilds.io/.

gkalpak committed:
ci: define common env vars in CircleCI job

mary-poppins commented:
You can preview 7379c23 at https://pr22810-7379c23.ngbuilds.io/.

mary-poppins commented:
You can preview 411f359 at https://pr22810-411f359.ngbuilds.io/.

alexeagle commented on scripts/ci/test-e2e-2.sh:
```diff
@@ -13,12 +13,6 @@ travisFoldStart "test.e2e.buildPackages"
   ./build.sh
```
yes, internally that integration test has `package.json` with ` "@angular/platform-server": "file:../../../dist/packages-dist/platform-server",` so it's really using the same formula as the other integration/* tests and should be moved, will do that

alexeagle committed:
test: move platform-server integration test

Now it lives in our standard location for tests against npm packages

alexeagle committed:
ci: remove travis e2e_2 job

The only remaining test can run in the first e2e travis shard.
This also removes the last thing needing bazel available on Travis.

mary-poppins commented:
You can preview 857d595 at https://pr22810-857d595.ngbuilds.io/.

IgorMinar commented on integration/run_tests.sh:
```diff
@@ -35,24 +63,23 @@ for testDir in $(ls | grep -v node_modules) ; do
-      rm -rf node_modules
+      #if $CI; then
+      #  trackPayloadSize "$testDir" "dist/*.js" true false "${basedir}/integration/_payload-limits.json"
+      #fi
```
so we are disabling the size tracking for cli hello world?
is this because we don't have credentials setup yet?
can you add a todo and explanation?
